### PR TITLE
Implement detail view

### PR DESCRIPTION
**Pull request by YKo20010**
#18 #19

### CODE REVIEW

File: acai-ios/CollectionViewCells/QuantitySelectionCollectionViewCell.swift [hunk 1]
```diff
@@ -21,14 +22,29 @@ class QuantitySelectionCollectionViewCell: UICollectionViewCell {
...
         self.backgroundColor = .white
         
+        border = UIView()
+        border.backgroundColor = .clear
+        border.layer.borderColor = UIColor.lineGray.cgColor
+        border.layer.borderWidth = 1
+        border.layer.cornerRadius = CGFloat(borderHeight/2)
+        contentView.addSubview(border)
+        
+        border.snp.makeConstraints { make in
```
Comment (young-k) [inline acai-ios/CollectionViewCells/QuantitySelectionCollectionViewCell.swift]: can this be moved to a `setupConstraints` method?

**Review by young-k — COMMENTED** [review]
This PR should be titled `Implemented DetailView`, or something w/o your name!

File: acai-ios/BowlOrderDetailViewController.swift [hunk 2]
```diff
@@ -39,6 +39,7 @@ class BowlOrderDetailViewController: UIViewController {
     let backButtonLeadingOffset = 18
     let backButtonHeight = 15
     let backButtonWidth = 8.7
+    let backgroundGradientScaleHeight: CGFloat = 3/5
```
Comment (kevinchan159) [inline acai-ios/BowlOrderDetailViewController.swift]: can we change to `0.6`

**Review by kevinchan159 — DISMISSED** [review]
lgtm!

**Review by kevinchan159 — DISMISSED** [review]

File: acai-ios/BowlOrderDetailViewController.swift [hunk 3]
```diff
@@ -36,21 +34,29 @@ class BowlOrderDetailViewController: UIViewController {
...
+        navigationController?.navigationBar.shadowImage = UIImage()
+        navigationController?.navigationBar.isTranslucent = true
+
+        navigationController?.navigationBar.tintColor = .white
+        navigationController?.navigationBar.titleTextAttributes = [
+            .font: UIFont.avenirNextBold.withSize(24),
+            .foregroundColor: UIColor.white
+        ]
+
+        // TODO: change to endpoint request data
```
Comment (cornell-appdev) [inline acai-ios/BowlOrderDetailViewController.swift:57]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/cuappdev/acai-ios/blob/dd4ba848aed599ded7efa3a362e18908cb048466/acai-ios/BowlOrderDetailViewController.swift#L57"></span>TODOs should be resolved (change to endpoint request dat…).<br />
<code>todo</code> <code>BowlOrderDetailViewController.swift:57</code></td>
    </tr>
  </tbody>
</table>

**Comment by cornell-appdev** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">SwiftLint not installed, download from https://github.com/realm/SwiftLint</td>
    </tr>
  </tbody>
</table>

### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
BowlOrderDetailViewController.swift | 57 | TODOs should be resolved (change to endpoint request dat...). (todo)


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Pull request merged**